src/en/space-station-14/departments/service/proposals/gnome-ghost-role.md

```diff
@@ -0,0 +1,85 @@
+# Botany helper gnomes
+
+| Designers      | Implemented | GitHub Links | Sprites |
+| -------------- | ----------- | ------------ |---------
+| BITTERLYNX     | :x: No      | https://github.com/space-wizards/space-station-14/pull/26941  | DanOftheE |
+
+# Botany helper gnomes
+
+Have YOU ever needed to garden AND needed to grab a drink at the bar? 
+Have YOU ever needed to fight mice on their own terf?
+Do YOU need someone to save you when the air is poisoned but nobody can help?
+Do _YOU_ need something to do once your soul is free floating and ready for harvest?
+
+Never fear! Nanotransen Reaserch and Devopment brings you a breakthrough in both biotech and anomaly studies! 
+THE GNOME! The best helper that botany can ask for! being small enough to get into those TIGHT spaces to do
+tough jobs we gaurentee you will be happy* for their help.
+
+(*Any defects are the responsibility of the station botanist)
+(rumors of gnomes commiting Homicide are false and defamitory. There is no Gnome king)
```

Comment:
something like this should be in the overview, not outside of it, also use direct language without asking and answering questions, be concise (same for the rest of the doc)